src/CMakeLists.txt

```diff
@@ -172,6 +172,7 @@ endif()
 
 if(WITH_ZLIB)
   find_package(ZLIB REQUIRED)
+  target_include_directories(rdkafka PUBLIC ${ZLIB_INCLUDE_DIRS})
```

Comment:
This is called .._DIRS while OPENSSL.. has .._DIR.
Are these standard names for these libraries? If not we should probably use the same form, _DIR or _DIRS.

Reply:
The standard name is `...DIRS`, but the OpenSSL find module is apparently inconsistent, and just uses `_DIR`. Compare:

  * https://cmake.org/cmake/help/latest/module/FindOpenSSL.html
  * https://cmake.org/cmake/help/latest/module/FindZLIB.html

AFAICT there just is literally no `OPENSSL_INCLUDE_DIRS` variable.

Reply:
Okay, thanks fort he clarification 👍 

Can you add a docker-based build script to packaging/tools/build-mingw.sh ?

Reply:
Sorry, wrong PR, nevermind the buildscript!